src/views/TheRoom.vue

```diff
@@ -92,6 +92,7 @@ export default class TheRoom extends mixins(RoomSocket) {
   }
   isDragEnabled = false
   isMapChanging = false
+  entityPanelOpen = true
```

Comment:
do we want isEntityPanelOpen prefix here?

Reply:
yes we do